test/base.test.js

```diff
@@ -36,9 +36,8 @@ test('should save the session properly', async (t) => {
       const keys = Object.keys(session)
 
       // Only storing three keys: cookie, encryptedSessionId and test
-      t.equal(keys.length, 3)
+      t.equal(keys.length, 2)
       t.ok(session.cookie)
-      t.ok(session.encryptedSessionId)
```

Comment:
this is still here, only via the getter, no? so `keys.length` is decreased, but this assertion would still work

Reply:
You are right.. lol. We need to check if it is not in the store saved

Reply:
created a TestStore, which uses JSON.stringify and parse to throw away the symbol keys.